src/components/App.tsx

```diff
@@ -40,6 +40,62 @@ export default function App() {
     setCurrentWallet(null)
   }, [setCurrentWallet])
 
+  const router = createBrowserRouter(
+    createRoutesFromElements(
+      <Route
+        id="base"
+        element={
+          <>
+            <Navbar />
+            <rb.Container as="main" className="py-4 py-sm-5">
+              {sessionConnectionError && (
+                <rb.Alert variant="danger">
+                  {t('app.alert_no_connection', { connectionError: sessionConnectionError.message })}.
+                </rb.Alert>
+              )}
+
+              <Layout />
+            </rb.Container>
+            <Footer />
+          </>
+        }
+      >
+        {/**
+         * This sections defines all routes that can be displayed, even if the connection
+         * to the backend is down, e.g. "create-wallet" shows the seed quiz and it is important
+         * that it stays visible in case the backend becomes unavailable.
+         */}
+        <Route id="create-wallet" path={routes.createWallet} element={<CreateWallet startWallet={startWallet} />} />
+        {/**
+         * This section defines all routes that are displayed only if the backend is reachable.
+         */}
+        {!sessionConnectionError && (
+          <>
+            <Route
+              id="wallets"
+              path={routes.home}
+              element={<Wallets currentWallet={currentWallet} startWallet={startWallet} stopWallet={stopWallet} />}
+            />
+            {currentWallet && (
+              <>
+                <Route id="wallet" path={routes.wallet} element={<MainWalletView />} />
+                <Route id="jam" path={routes.jam} element={<Jam />} />
+                <Route id="send" path={routes.send} element={<Send />} />
+                <Route id="earn" path={routes.earn} element={<Earn />} />
+                <Route id="receive" path={routes.receive} element={<Receive />} />
+                <Route id="settings" path={routes.settings} element={<Settings stopWallet={stopWallet} />} />
+              </>
+            )}
+            <Route id="404" path="*" element={<Navigate to={routes.home} replace={true} />} />
```

Comment:
It seems like the 404 redirect does not work, here is what is shown to me when I navigate to a route that doesn't exist:

![Screenshot 2022-09-28 at 14 46 20](https://user-images.githubusercontent.com/109058/192782192-f5c07314-f64d-4012-aa3a-c8dda011dd04.png)

Reply:
Hmm... cannot reproduce currently. If I navigate to an non-existing route by typing it into the address bar, it will take me to `/` (as expected).
Sorry for my stupid question, but how did you navigate to the non-existing route?

I have only seen this meesage, when I abort `npm start`.. 
.. did you execute `npm install`?

Reply:
> (unless it's something I'm doing wrong on my side)

> did you execute `npm install`?

🤦‍♂️

Reply:
My bad. Reminder for myself: Always include a "How to test" section in PRs.

Reply:
I have seen this screen now myself for a very brief moment when reloading a page. Will look into it!